dh_collections_prism_collection_community.json

```diff
@@ -956,29 +953,6 @@
     "collection_id": "Series IV. Photographs",
     "community_id": "g-v-black-papers"
   },
-  "ce39f2be-9a64-4717-973b-ff531c2a93ee": {
-    "collection_id": "",
-    "community_id": "communication-bridge"
-  },
-  "ed35d953-d035-44a7-bacb-0edf1c8006a6": {
-    "community_id": "endowed-professorship-biographies"
-  },
-  "a97853c7-82d3-408c-95d8-c72a04acb41f": {
-    "collection_id": "James A. Conner",
-    "community_id": "12th General Hospital Collection"
-  },
-  "b0a1fa15-eb29-416f-b452-e81b39f0f26b": {
-    "collection_id": "Michael L. Mason Collection",
-    "community_id": "12th-general-hospital-collection"
-  },
-  "2f75r807r": {
-    "collection_id": "Portraits",
-    "community_id": "history-of-feinberg-school-of-medicine"
-  },
-  "areybook": {
-    "collection_id": "Northwestern University Medical School, 1859-1979: A pioneer in educational reform",
-    "community_id": "history-of-feinberg-school-of-medicine"
-  },
```

Comment:
Those were duplicated. Perhaps double-check if this is masking a deeper issue (some collections that should have been there?).

Reply:
Same question as below.

Reply:
Okay, I found the duplicate entry in `dh_collections_prism_collection_community.json` (file within DigitalHub repo). I'm going to run a script to check for other duplicates too.

I'm surprised that using `JSON.parse(File.read(DH_COLLECTIONS_TO_PRISM_COLLECTION_COMMUNITY_JSON))` in the converter had issues with duplicates since it creates a hash which should not allow for duplicate keys.